mmv1/compiler.rb

```diff
@@ -98,6 +103,11 @@
 raise 'Option -o/--output is a required parameter' if output_path.nil?
 raise 'Option -e/--engine is a required parameter' if provider_name.nil?
 
+if openapi_generate
+  # Test write OpenAPI --> YAML
+  OpenApiParser::Parser.new('open_api_parser/specs/*', 'products/demo').run
```

Comment:
Should we output directly to `products/` or is it important that we use a subfolder?

Reply:
It doesn't output the product info yet, so it needs to be wrapped in a fake product folder. When we add support for product generation it can make the whole product.yaml & folder from an OpenAPI spec

Reply:
Thanks! Do you mind putting that in a comment?